class.jetpack-cli.php

```diff
@@ -183,7 +183,7 @@ public function test_connection( $args, $assoc_args ) {
 	 */
 	public function disconnect( $args, $assoc_args ) {
 		if ( ! Jetpack::is_active() ) {
-			WP_CLI::error( __( 'You cannot disconnect, without having first connected.', 'jetpack' ) );
+			WP_CLI::success( __( 'The site is not currently connected, so nothing to do!', 'jetpack' ) );
```

Comment:
Does `::success()` exit?

Reply:
It does not. 66e654a9eb3cc811ac6154f77f967a9577822cf9